plugins/Morpheus/stylesheets/base/colors.less

```diff
@@ -26,7 +26,7 @@
 // Variable pattern: @color-<color>-<modifier>
 @color-black-matomo:               #0d0d0d;
 @color-blue-matomo:                #4183C4;
-@color-red-matomo:                 #d4291f;
```

Comment:
The (old) color is still referenced by hex value in some places:

- [/plugins/Morpheus/stylesheets/main.less](https://github.com/matomo-org/matomo/blob/418eaa66f9fbfc8426c6fee21f617fa9b7d790c4/plugins/Morpheus/stylesheets/main.less#L738)
- [plugins/SegmentEditor/stylesheets/segmentation.less](https://github.com/matomo-org/matomo/blob/418eaa66f9fbfc8426c6fee21f617fa9b7d790c4/plugins/SegmentEditor/stylesheets/segmentation.less#L185)

And for completeness perhaps the values in [/plugins/ExampleTheme/stylesheets/theme.less](https://github.com/matomo-org/matomo/blob/418eaa66f9fbfc8426c6fee21f617fa9b7d790c4/plugins/ExampleTheme/stylesheets/theme.less#L17C1-L17C1)?

Or was any of those intentionally kept with the old color?

_There is also [plugin-LoginLdap/.../TestableField.less](https://github.com/matomo-org/plugin-LoginLdap/blob/0c283b7956de4081488da28e37250d499c342bd4/vue/src/TestableField/TestableField.less#L5) that might want to receive the same treatment?_

Reply:
Guess those slipped through. I've updated the PR